core/controller/src/main/scala/whisk/core/controller/RestAPIs.scala

```diff
@@ -160,7 +160,7 @@ protected[controller] class RestAPIVersion(apipath: String, apiversion: String)(
             sendCorsHeaders {
                 (pathEndOrSingleSlash & get) {
                     complete(OK, info)
-                } ~ authenticate(basicauth) {
+                } ~ (authenticate(basicauth) | authenticate(certificateAuth)) {
```

Comment:
Because nginx.conf.j2 has below configuration
```
 proxy_set_header X-SSL-Subject $ssl_client_s_dn;
```
It will protect passing `x-ssl-subject` http header from nginx to controller which means the controller can't receive the `x-ssl-subject` from nginx.

so if use blow curl cmd, the controller can't receive the `x-ssl-subject` http header
```
curl -v -X  GET \
--insecure \
--header 'Accept: application/json' \
--header 'X-SSL-Subject: CN=guest,O=OpenWhisk,L=Yorktown,ST=NY,C=US' \
--header 'namespace: guest' \
https://<nginx>/api/v1/namespaces/guest/actions
There was an internal server error.
```
So  i think current codes is ok